cli/src/main/java/de/jplag/cli/options/CliOptions.java

```diff
@@ -99,6 +99,9 @@ public static class Advanced {
 
         @Option(names = "--csv-export", description = "Export pairwise similarity values as a CSV file.")
         public boolean csvExport = false;
+
+        @Option(names = "--overwrite", description = "If this option is specified and a result file already exists it will be overwritten.")
```

Comment:
I would change this text to be more online with other flags. Meaning: Just state what it does without the `If this option is specified`